src/main/java/de/cronn/validation_files_diff/helper/PsiTestNameUtils.java

```diff
@@ -0,0 +1,28 @@
+package de.cronn.validation_files_diff.helper;
+
+import com.intellij.psi.PsiClass;
+import com.intellij.psi.PsiMethod;
+
+public class PsiTestNameUtils {
```

Comment:
Probably you used `de.cronn.assertions.validationfile.util.TestNameUtils` as template for creating this class. Is there a change to change `PsiTestNameUtils` in such a way to use `de.cronn.assertions.validationfile.util.TestNameUtils`? Or maybe add a reference as comment, so that there is a hint where to look in case that the underlying class changes and things break?

Reply:
I fear not. `TestNameUtils` works with actual `Class` from java lib, while intellij has its `PsiClass` which have no common denominator. 
There is also no way in converting the PsiClass to an actuall class since this would mean we would need to compile and load the test classes in our jvm.